src/server/verification/verificationAPI.js

```diff
@@ -102,7 +103,8 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
         res.json({ success: true, sessionToken })
       } catch (exception) {
         const { message } = exception
-        log.error('generating enrollment session token failed:', message, exception, { user })
+
+        logException(log, 'generating enrollment session token failed:', message, exception, { user })
```

Comment:
should be error

Reply:
i've checked -  yes, here only something unexpected such as the timeout could be thrown, so let it be logged as .error()